frontend/src/components/Service.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    
+</template>
+
+<template>
+  <v-container>
+    <v-layout
+      text-xs-center
+      wrap
+    >
+
+       <div class="card my-5">
+        <div class="card-body">
+            <h5 class="text-center card-title">{{ service.title }}</h5>
+        </div>
+    </div>
+
+    </v-layout>
+  </v-container>
+</template>
+
+
+<script>
+export default{
```

Comment:
Again, look at `src/views/Home.vue` to see how to set this up using TypeScript. Also look at the vue typescript tutorial I sent on slack. Make sure your script tag has the `lang="ts"` attribute